backend/main.py

```diff
@@ -74,13 +75,84 @@ def SignIn():
             #One-way hash inputted password to see if it matches db records. If no match, raises exception
             verify_result = salted_hasher.verify(db_hashed_passwd, SignIn_data.get("password"))
             #If passed without exceptions:
-            response_object["username"] = db_username
-            response_object["email"] = db_email
-            response_object["message"] = "User account found and logged in"
+            response["username"] = db_username
+            response["email"] = db_email
+            response["message"] = "User account found and logged in"
         except:
             cursor.close()  
-            response_object["message"] = "User account not found"
-    return jsonify(response_object)
+            response["message"] = "User account not found"
+    return jsonify(response)
+
+@app.route("/PredPrey/AlterPresets", methods=["POST"])
+def add_PredPrey_preset():
+    response = {"server status": "success"}
```

Comment:
what if the response isn't a success?

Reply:
Then Flask chucks an error and the catch block's activated within the client-side javascript. The success part refers to reaching the server successfully